source/_integrations/huawei_lte.markdown

```diff
@@ -75,8 +75,29 @@ entities varies by device model and firmware version.
 
 {% include integrations/config_flow.md %}
 
-Unauthenticated mode and default list of notification recipient phone
-numbers can be set using the integration's configuration options.
+{% configuration_basic %}
+URL:
+  description: Base URL to the API of the router. Typically, something like `http://192.168.X.1` where `X` is for example `1`, `8`, or `100`. This is the beginning of the location shown in a browser when accessing the router's web interface.
+Verify SSL certificate:
+  description: Whether to verify the SSL certificate of the router when accessing it. Applicable only if the router is accessed via HTTPS, i.e. if the configured URL starts with `https://`.
```

Comment:
```suggestion
  description: Whether to verify the SSL certificate of the router when accessing it. Applicable only if the router is accessed via HTTPS. For example, if the configured URL starts with `https://`.
```

Reply:
Thanks for the tweaks! "For example" is not accurate for this though, "i.e." was. If we want to expand it, "in other words" would be a better replacement.

Reply:
https://github.com/home-assistant/home-assistant.io/pull/38497

Reply:
Oh, right. Thank you for the catch, @scop :heart: